aha-table.html

```diff
@@ -429,15 +429,23 @@
       var scrollBody = Polymer.dom(aha.root).querySelector('.scroll-body.aha-table');
       var rowHeight = 42;
 
+      var closestPageSize;
```

Comment:
Why it needs to be done: http://stackoverflow.com/questions/6033821/do-i-need-to-remove-event-listeners
Callback hook in Polymer: `detached`, https://www.polymer-project.org/1.0/docs/devguide/registering-elements#lifecycle-callbacks
About removeEventListener: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener